api/ai/priority_algorithm/priority/priority.py

```diff
@@ -37,6 +37,9 @@ def satisfaction(self, students: List[Student]) -> float:
     def satisfied_by(self, students: List[Student]) -> bool:
         count = 0
         for student in students:
+            if student.attributes is None:
```

Comment:
it's never None, this should be impossible and if it happens, I want it to throw an error